pennylane_lightning/core/src/simulators/lightning_kokkos/catalyst/LightningKokkosSimulator.hpp

```diff
@@ -174,6 +174,7 @@ class LightningKokkosSimulator final : public Catalyst::Runtime::QuantumDevice {
     auto CacheManagerInfo()
         -> std::tuple<std::size_t, std::size_t, std::size_t,
                       std::vector<std::string>, std::vector<ObsIdType>>;
+    auto GenerateSamples(const size_t shots) -> std::vector<size_t>;
```

Comment:
This seems like it should be a private method if it's purpose is being a helper function

Reply:
good point

Reply:
done